.github/workflows/collector.yml

```diff
@@ -0,0 +1,24 @@
+name: cobrowse-evidence-flow
```

Comment:
Not sure this name is appropriate – copy paste error?

Reply:
Ha. Yeah, I should fix that in my template file, as well as here.